src/main/java/com/mgmtp/radio/respository/youtube/YouTubeRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.mgmtp.radio.respository.youtube;
+
+import com.mgmtp.radio.domain.youtube.YouTubeVideo;
+import org.springframework.data.mongodb.repository.MongoRepository;
+
+public interface YouTubeRepository extends MongoRepository<YouTubeVideo, String> {
+    YouTubeVideo findByTitle(String title);
```

Comment:
Why we should save youtube? The terms and conditions do not allow us save resources from youtube

Reply:
This is a redundant file. I removed it.